custom_model_runner/datarobot_drum/drum/main.py

```diff
@@ -92,11 +92,11 @@ def signal_handler(sig, frame):
             try:
                 loader = RuntimeParametersLoader(options.runtime_params_file, options.code_dir)
                 loader.setup_environment_variables()
-                if RuntimeParameters.has("CUSTOM_MODEL_WORKERS"):
-                    options.max_workers = RuntimeParameters.get("CUSTOM_MODEL_WORKERS")
             except RuntimeParameterException as exc:
                 print(str(exc))
                 exit(255)
+        if RuntimeParameters.has("CUSTOM_MODEL_WORKERS"):
+            options.max_workers = RuntimeParameters.get("CUSTOM_MODEL_WORKERS")
```

Comment:
I presume this is meant to be a `numeric` type -- if so, shouldn't this be cast as an `int()` because in the platform all numerics are floats

Reply:
It works even if user define float like "2.1"  https://staging.datarobot.com/registry/custom-model-workshop/66e9709bb28fc936864e52e2/test.  But ideally I am thinking about int runtime parameters to not allow user define float value at all.  I can update to options.max_workers = init(RuntimeParameters.get("CUSTOM_MODEL_WORKERS")) if it more safe/clear for us. Does it cause any issues today ?

Reply:
I just tested in my own model and it looks like it works as is. I was just concerned because I ran into a case recently where something didn't support "1.0" so I needed to cast it to an int first 🤷‍♂️. Anyway, I agree what you need more is a constraint on the UI side to say only allow integer values.

Reply:
We never added ints to runtime params because we are trying to keep things simple and there haven't been many cases we found where we really care about an int vs a float.